PyGame_Project/MVC/Controller/CLI_controller.py

```diff
@@ -348,6 +348,9 @@ def active_game_commands(userInput):
       high_score_current_puzzle()
       space_out()
       return True
+    
+    case "/shareimage":
+      generate_new_image()
 
```

Comment:
return True

currently sends you back to main menu when typing /shareimage in active game

Reply:
(add "return True" to line 354)

Reply:
Also after you change that, can you change:

line 248 from return False to return True
line 250 from return True to return False

future ben thanks you